distribution/kubernetes/vector-agent/configmap.yaml

```diff
@@ -0,0 +1,42 @@
+---
+# This file has been generated by `helm template vector vector/vector` from vector/templates/configmap.yaml. Please re-run `make generate-kubernetes-manifests` rather than modifying this file manually.
```

Comment:
I'd suggest marking these files as "generated" so that diffs are easier to scan for non-generated changes. Granted, with this initial PR, reviewing the generated files _is_ useful 😄 

https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

Reply:
I feel like we may always want to see the diffs here to ensure we're not introducing breaking changes accidentally, but we can make that change later as well.

Reply:
TIL about that option! Also agree that the diff can be quite valuable.